db/warehouse/migrate/20220621181125_copy_health_housing_navigator_to_client.rb

```diff
@@ -0,0 +1,10 @@
+class CopyHealthHousingNavigatorToClient < ActiveRecord::Migration[6.1]
+  def up
+    PaperTrail.enabled = false # Migration fails with can't find 'versions' table w/ papertrail enabled.
+
+    Health::Patient.find_each do |patient|
+      client = GrdaWarehouse::Hud::Client.find(patient.client_id)
+      client.update!(health_housing_navigator_id: patient.housing_navigator_id) if client.present?
+    end
```

Comment:
You may want:
```suggestion
    Health::Patient.find_each do |patient|
      client = GrdaWarehouse::Hud::Client.find(patient.client_id)
      next unless client.present?
      
      client.update!(health_housing_navigator_id: patient.housing_navigator_id) if client.present?
    end
```

Reply:
Do you think it is clearer with the guard expression vs the trailing `if`? I'm fine with either way. (But it seems strange to have both)

Reply:
oh, sorry, missed the trailing if.  That's totally fine.